espflash/tests/scripts/save-image_write-bin.sh

```diff
@@ -12,7 +12,7 @@ if [[ ! $result =~ "Image successfully saved!" ]]; then
     exit 1
 fi
 echo "Writing binary"
-result=$(timeout 45 espflash write-bin --monitor 0x0 app.bin --non-interactive 2>&1)
+result=$(timeout 90 espflash write-bin --monitor 0x0 app.bin --non-interactive 2>&1)
```

Comment:
This looks like I'm trying to hide the issue, but I believe that now `verify` and `skip` are respected, they are now used by default. This adds time to CI, and on some chips this takes longer than 45s.

Reply:
The same for the timeout in the hi.yml